ee/ucp/admin/configure/configure-rbac-kube.md

```diff
@@ -15,47 +15,46 @@ UCP 3.0 used its own role-based access control (RBAC) for Kubernetes clusters. N
 
 Kubernetes RBAC is turned on by default for Kubernetes clusters when customers upgrade to UCP 3.1. See [Using RBAC Authorization](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) in the v1.11 documentation for more information about Kubernetes role-based access control.
 
-Starting with UCP 3.1, Kubernetes and Swarm roles have separate views. You can view all the roles for a particular cluster under **Access Control** then **Roles**. Select Kubernetes or Swarm to view the specific roles for each.
+Starting with UCP 3.1, Kubernetes and Swarm roles have separate views. You can view all of the roles for a particular cluster under **Access Control** > **Roles**. Select **Kubernetes** or **Swarm** to view the specific roles for each.
 
 ## Creating roles
 
-You create Kubernetes roles either through the CLI using `kubectl` or through the UCP web interface.
-
-To create a Kubernetes role in the UCP web interface:
+You can create Kubernetes roles either through the CLI using [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) or through the UCP web interface. The following procedure uses the UCP web interface.
```

Comment:
I suggest placing "either" after the first "through" and removing the second "through".

"You can create Kubernetes roles through either the CLI using [kubectl] or the UCP web interface (the procedure that follows)."